friends.gemspec

```diff
@@ -9,7 +9,8 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Jacob Evelyn"]
   spec.email         = ["jacobevelyn@gmail.com"]
   spec.summary       = "Spend time with the people you care about."
-  spec.description   = "Spend time with the people you care about. Introvert-tested. Extrovert-approved."
+  spec.description   = "Spend time with the people you care about.
```

Comment:
Could you change this to:

```ruby
"Spend time with the people you care about. "\
"Introvert-tested. Extrovert-approved."
```

...?

That way the string won't have a bunch of whitespace in it.